source/includes/_documentation.md

```diff
@@ -4336,9 +4336,304 @@ There is two different checkboxes in Angular: Basic and Configurable:
+**getAction(Field field, Object element, Object parent)** | defines the specifics of how form elements will be obtained for verification and checks | void
+**isDisplayed()** | check that form is displayed | boolean
+**isValid()** | check that form is valid | boolean
+**login()** | clicks “login” Button or ”loginButton” | boolean
+
+
+![Input with hints](../images/angular/input_with_hints.png)
+```html
+<input _ngcontent-ohc-c261="" matinput="" id="inputs_hints_message" maxlength="256" placeholder="Ex. I need help with..." class="mat-input-element mat-form-field-autofill-control ng-tns-c94-53 cdk-text-field-autofill-monitored" aria-describedby="inputs_hints_text inputs_hints_counter" aria-invalid="false" aria-required="false">
+<mat-hint _ngcontent-ohc-c261="" id="inputs_hints_text" class="mat-hint ng-tns-c94-53"><strong _ngcontent-ohc-c261="">Don't disclose personal info</strong></mat-hint>
+```
+
+|Method | Description | Return Type
+--- | --- | ---
+**focus()** | place cursor within the text field | void
+**sendKeys(String value)** | send text to the text field | void
+**setText(String value)** | set text | void
+**input(String value)** | set text | void
+**getText()** | returns text from the text field | String
+**getValue()** | returns text from the text field | String
+**clear()** | clears the text field | void
+**is()** | property that returns object for work with assertions | TextAssert
+**placeholder()** | returns value of placeholder attribute | String
+
+
+![Input with prefixes and suffixes](../images/angular/input_with_prefixes_and_suffixes.png)
+```html
+<span _ngcontent-ohc-c262="" matprefix="" id="inputs_prefixes" class="ng-tns-c94-54">+1 &nbsp;</span>
+<input _ngcontent-ohc-c262="" type="tel" matinput="" id="inputs_prefixes_suffixes_phone" placeholder="555-555-1234" class="mat-input-element mat-form-field-autofill-control ng-tns-c94-54 cdk-text-field-autofill-monitored" aria-invalid="false" aria-required="false">
+<mat-icon _ngcontent-ohc-c262="" role="img" matsuffix="" id="inputs_suffixes" class="mat-icon notranslate material-icons mat-icon-no-color ng-tns-c94-54" aria-hidden="true">edit</mat-icon>
+```
+
+|Method | Description | Return Type
+--- | --- | ---
+**focus()** | place cursor within the text field | void
+**sendKeys(String value)** | send text to the text field | void
+**setText(String value)** | set text | void
+**input(String value)** | set text | void
+**getText()** | returns text from the text field | String
+**getValue()** | returns text from the text field | String
+**clear()** | clears the text field | void
+**is()** | property that returns object for work with assertions | TextAssert
+**placeholder()** | returns value of placeholder attribute | String
+
+
+#### <a href="https://github.com/jdi-testing/jdi-light/tree/angular/jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests" target="_blank">Here you can find Input tests</a>
```

Comment:
Here should be a link to master. 
Also, I know that we have "Here you can find ..." links in Angular module, but I think it'd be better to add link text "Java tests examples" as it is here https://jdi-docs.github.io/jdi-light/?java#label